optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/importing/ZeebeConstants.java

```diff
@@ -16,13 +16,29 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class ZeebeConstants {
 
+  // zeebe export
   public static final String ZEEBE_RECORD_TEST_PREFIX = "zeebe-record";
-  public static final String ZEEBE_DEFAULT_TENANT_ID = "<default>";
-  public static final String ZEEBE_DEFAULT_TENANT_NAME = "Default Tenant";
   public static final String ZEEBE_OPENSEARCH_EXPORTER =
       "io.camunda.zeebe.exporter.opensearch.OpensearchExporter";
   public static final String ZEEBE_ELASTICSEARCH_EXPORTER =
       "io.camunda.zeebe.exporter.ElasticsearchExporter";
+
+  // variable types
+  public static final String VARIABLE_TYPE_OBJECT = "Object";
+  public static final String VARIABLE_TYPE_JSON = "Json";
+  public static final String VARIABLE_SERIALIZATION_DATA_FORMAT = "serializationDataFormat";
+
+  // incident types
+  public static final String FAILED_JOB_INCIDENT_TYPE = "failedJob";
+  public static final String FAILED_EXTERNAL_TASK_INCIDENT_TYPE = "failedExternalTask";
+
+  // tenant
+  public static final String ZEEBE_DEFAULT_TENANT_ID = "<default>";
+  public static final String ZEEBE_DEFAULT_TENANT_NAME = "Default Tenant";
   public static final TenantDto ZEEBE_DEFAULT_TENANT =
       new TenantDto(ZEEBE_DEFAULT_TENANT_ID, ZEEBE_DEFAULT_TENANT_NAME, ZEEBE_DATA_SOURCE);
+
+  // flownode types
+  public static final String FLOW_NODE_TYPE_USER_TASK = "userTask";
+  public static final String FLOW_NODE_TYPE_MI_BODY = "multiInstanceBody";
```

Comment:
Just thinking, is _this_ the reason why we have that other heatmap bug? i.e. in C7 we can detect when flownodes are part of a subprocess, but in C8 can't? If so, maybe using this approach again would even be the better solution. Wdyt?

Reply:
You mean the issue about Optimize not hiding the hidden/collapsed subprocess data correctly in the heatmap? Didnt even know this doesnt affect C7. Looking at the usages, `FLOW_NODE_TYPE_MI_BODY` is only used in ModelElementFilterQueries to exclude data from that type of node. I'm not deep enough in the context of the issue yet to know whether that would be relevant but lets keep it in mind